lib/packages/shared-types/events/new-chip-submission.ts

```diff
@@ -0,0 +1,42 @@
+import { z } from "zod";
+import {
+  attachmentArraySchema,
+  attachmentArraySchemaOptional,
+} from "../attachments";
+
+export const feSchema = z.object({
```

Comment:
@mdial89f Would you mind working off of this file and fill in the blanks? I'm not sure what goes where with your structure.